paywall/src/__tests__/unlock.js/CheckoutUIHandler.test.ts

```diff
@@ -159,7 +162,13 @@ describe('CheckoutUIHandler', () => {
 
     const data: RelayedMessageTestType = [
       ['UPDATE_ACCOUNT', PostMessages.UPDATE_ACCOUNT, fakeAccount],
-      ['UPDATE_ACCOUNT_BALANCE', PostMessages.UPDATE_ACCOUNT_BALANCE, '123'],
+      [
+        'UPDATE_ACCOUNT_BALANCE',
+        PostMessages.UPDATE_ACCOUNT_BALANCE,
+        {
+          eth: '123',
```

Comment:
This test was still in the old balance format. Fixed it while I was in there, but it's not critical since this test just verifies that it passes through without touching the data.